app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import abstractmethod, ABC
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating() -> None:
+        pass
+
+    @abstractmethod
+    def player_info() -> None:

```

Comment:
The `player_info` method should include `self` as its first parameter, like so: `def player_info(self) -> None:`. This is necessary for instance methods in Python.